posix.mak

```diff
@@ -23,21 +37,40 @@ else
 	DOTLIB:=.a
 endif
 
-DFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -inline -w -Isrc -Iimport $(PIC)
-UDFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -w -Isrc -Iimport $(PIC)
 DDOCFLAGS=-conf= -c -w -o- -Isrc -Iimport -version=CoreDdoc
 
-CFLAGS=$(MODEL_FLAG) -O $(PIC)
+# Set CFLAGS
+CFLAGS=
+ifneq (,$(filter cc% gcc% clang% icc% egcc%, $(CC)))
```

Comment:
I didn't invent this but took the code from https://github.com/D-Programming-Language/phobos/blob/88fc63d8d3d47ded732e801d4ccacec21eccb8f2/posix.mak#L96? So why did this work with your phobos build? You can still set `CFLAGS` explicitly.